### DISCUSSION TITLE
Add collapsible tabs demo to the docs

### DISCUSSION THREAD

**Discussion by PedroBern** [software-mansion/react-native-reanimated#1656] [Show and tell]
Hi, thanks for this awesome lib

I've made a package to build a collapsible tab bar with reanimated v2, and would like to know if I can add it to the Example app. What do you think? If it's appreciated, I can send a PR :)

Here is my expo preview (I'm using the [react-native-tab-view](https://github.com/satya164/react-native-tab-view) example app as template):

- [react-native-collapsible-tab-view](https://github.com/PedroBern/react-native-collapsible-tab-view)
- [Expo preview](https://expo.io/@pedrobern/react-native-collapsible-tab-view-demos)

<a href="https://expo.io/@pedrobern/react-native-collapsible-tab-view-demos"><img src="https://api.qrserver.com/v1/create-qr-code/?size=400x400&data=exp://exp.host/@pedrobern/react-native-collapsible-tab-view-demos" height="200px" width="200px"></a>

Some demos:

|                                                     Default                                                      |                                                     Snap                                                      |                                                     DiffClamp                                                      |                                                    DiffClamp + Snap                                                     |
| :--------------------------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------: |
| <img src="https://github.com/PedroBern/react-native-collapsible-tab-view/raw/main/demo/default.gif" width="360"> | <img src="https://github.com/PedroBern/react-native-collapsible-tab-view/raw/main/demo/snap.gif" width="360"> | <img src="https://github.com/PedroBern/react-native-collapsible-tab-view/raw/main/demo/diffClamp.gif" width="360"> | <img src="https://github.com/PedroBern/react-native-collapsible-tab-view/raw/main/demo/diffClamp_snap.gif" width="360"> |

**Comment by karol-bisztyga** [comment]
Looks nice! Why don't you just create a PR then, that'd be a better place to discuss whether to merge it or not :)

**Reply by PedroBern** [reply]
ok then :) We are polishing the API, I will open the PR when it's ready :) thanks

**Reply by PedroBern** [reply]
@karol-bisztyga thanks for the response, I'm going to wait for the stable release to open the PR, because we are supporting expo. Everything is working on rc.0, but not on rc.3! 😅 

Also, thanks for the awesome work.

**Reply by karol-bisztyga** [reply]
Sure, thanks for contributing, looking forward to seeing this!
